packages/main/src/components/AnalyticalTable/ColumnHeader/ColumnHeaderModal.tsx

```diff
@@ -80,6 +92,9 @@ export const ColumnHeaderModal: FC<ColumnHeaderModalProperties> = (props) => {
     [column, popoverRef, onGroupBy]
```

Comment:
Add onSort to dependency array
```suggestion
    [column, popoverRef, onGroupBy, onSort]
```

Reply:
done